src/features/list-of-mentors/list-of-mentors.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import classNames from 'classnames';
+import { shallowEqual, useSelector } from 'react-redux';
+import { Card, Search, Button } from '../../components/index.js';
+import Icon from '../../icon.js';
+import { actions, searchMentorValue } from './redux/index.js';
+import { useActions } from '../../shared/hooks/index.js';
+import styles from './list-of-mentors.scss';
+import { dataList } from './mentors-dataList.js';
+
+export const ListOfMentors = () => {
+  // Search input
+  const { setSearchMentorValue } = useActions(actions);
+  const searchMentorName = useSelector(searchMentorValue, shallowEqual);
+
+  const handleSearch = (inputValue) => {
+    setSearchMentorValue(inputValue);
+  };
+
+  // Add a Mentor
+  const addMentor = () => {
+  };
+
+  // Mentors details
+  const cardDetails = (id) => {
+  };
+
+  // Edit Mentor's details
+  const cardEditing = (id) => {
+  };
+
+  const mentorsList = () => {
+    const listByMentorName = dataList.filter((mentor) => mentor.name.toUpperCase()
+      .includes(searchMentorName.toUpperCase()));
+
+    return listByMentorName.map((mentor) => (
+      <Card
+        key={mentor.uuid}
+        id={mentor.uuid}
+        button="Details"
+        onEdit={cardEditing}
+        onDetails={cardDetails}
+      > { mentor.name }
+      </Card>
```

Comment:
Replace prop name of button on `buttonName="Details"` and add prop `iconName="Edit"`

Reply:
done